js/simpletextbox/Example.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+	import { onMount } from "svelte";
+
+	export let value: string;
+	export let type: "gallery" | "table";
+	export let selected = false;
+
+	let size: number;
+	let el: HTMLDivElement;
+
+	function set_styles(element: HTMLElement, el_width: number): void {
+		if (!element || !el_width) return;
+		el.style.setProperty(
+			"--local-text-width",
+			`${el_width < 150 ? el_width : 200}px`
+		);
+		el.style.whiteSpace = "unset";
+	}
```

Comment:
Hmm, forgot about this. Maybe we can figure out a way to move it into the Dataset.